app/models/organization.rb

```diff
@@ -0,0 +1,8 @@
+class Organization < ApplicationRecord
+  has_one_attached :file
+
+  has_many :courses
+  has_many :occupation_standards
+
+  validates :title, presence: true
```

Comment:
Let's also add uniqueness validation here.
```suggestion
  validates :title, presence: true, uniqueness: true
```